### PR TITLE
No tsinfer

### PR DESCRIPTION
**Pull request by petrelharp**
Ok, here's a version without time decomposition or tsinfer trees. Probably you should edit this version, @molpopgen ?

Also note the additional paragraph I put in the discussion.

### CODE REVIEW

File: treestats_paper.tex [hunk 1]
```diff
@@ -1458,6 +1354,15 @@ \section*{Discussion}
 calculation of branch statistics would provide a summary method
 that is not affected by this potentially important confounding factor.
 
+Many genomic datasets only contain genotype information at a subset of sites
```
Comment (molpopgen) [inline treestats_paper.tex]: This is good!

**Comment by molpopgen** [comment]
Looks good.  I like the bit re:ascertainment

File: treestats_paper.tex (hunk 1 above)
Comment (petrelharp) [reply treestats_paper.tex]: yes!!! realized that on the bike ride in this morning, talking to andy.

**Comment by molpopgen** [comment]
Related, but so hard to think about: imputation!
__
Kevin Thornton
Associate Professor
Ecology and Evolutionary Biology
UC Irvine
www.molpopgen.org
github.com/molpopgen
github.com/ThorntonLab

**Pull request merged**